src/megaclient.cpp

```diff
@@ -6108,10 +6111,15 @@ int MegaClient::readnodes(JSON* j, int notify, putsource_t source, NewNode* nn,
                     {
                         app->reload("Node inconsistency");
 
-                        int creqtag = reqtag;
-                        reqtag = 0;
-                        sendevent(99437, "Node inconsistency");
-                        reqtag = creqtag;
+                        static bool reloadnotified = false;
```

Comment:
Good catch! I guess we send event for every node detected as inconsistent before actually start a reload. But... after a reload, will we stop notifying future inconsistencies?

Reply:
Reloads are done after the app restarts, so there's no such issue :)